pom.xml

```diff
@@ -76,6 +76,10 @@
     <it.windows>false</it.windows>
     <configuration-as-code.version>1.14</configuration-as-code.version>
     <powershell.version>1.3</powershell.version>
+    <google-oauth.version>1.0.0-SNAPSHOT</google-oauth.version>
+    <google.guava.version>20.0</google.guava.version>
+    <gcp-plugin-core.version>0.2.0-SNAPSHOT</gcp-plugin-core.version>
```

Comment:
I'll update to remove the "-SNAPSHOT" from both versions once those releases are ready.